### DISCUSSION TITLE
How to import LazyVim as type in configuration?

### DISCUSSION THREAD

**Discussion by seflue** [LazyVim/LazyVim#3108] [Q&A] [answered]
The documentation shows a lot of snippets with `LazyVim` like in the example for [nvim-lspconfig](https://www.lazyvim.org/plugins/lsp#nvim-lspconfig), but it doesn't actually shows, where the symbol `LazyVim` is coming from or how it is imported. In the starter example it is not mentioned. Can someone help me out, because I didn't find any hint while skimming through the docs.

**Comment by dpetka2001** [comment, marked as answer]
You can find it [here](https://github.com/LazyVim/LazyVim/blob/530e94a9fa19577401e968a9673282c3d79f01e3/lua/lazyvim/config/init.lua#L1-L7). It's just a global variable used for abbreviating `require("lazyvim.util")`. And `LazyVim.config` is just an abbreviation for accessing the `lazyvim.config` module (instead of writing `require("lazyvim.config")`).
